plugins/modules/ndo_fabric_span_session.py

```diff
@@ -0,0 +1,698 @@
+- The O(destination_epg.epg) must exist before using it with this module in your playbook.
+  Use M(cisco.mso.mso_schema_template_anp_epg) to create the EPG.
+- The O(destination_port_channel.port_channel) must exist before using it with this module in your playbook.
+  Use M(cisco.mso.ndo_port_channel_interface) to create the Fabric resource port channel interface.  
+seealso:
+- module: cisco.mso.ndo_template
+- module: cisco.mso.mso_schema_template_anp_epg
+- module: cisco.mso.ndo_port_channel_interface
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create EPG - Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_epg
+    destination_epg:
+      epg:
+        schema: ansible_test
+        template: Template1
+        anp: Anp1
+        name: EPG1
+      destination_ip: "1.1.1.1"
+      source_ip_prefix: "2.2.2.2"
+    state: present
+  register: add_epg_span_session
+
+- name: Create Port - Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_port
+    destination_port:
+      port:
+        node: 101
+        path: "eth1/1"
+    state: present
+
+- name: Create Port Channel - Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_pc
+    destination_port_channel:
+      port_channel:
+        template: ansible_test
+        name: ansible_test_resource_pc_1
+    state: present
+
+- name: Update/Change EPG - Fabric SPAN Session to Port Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_epg
+    destination_port:
+      port:
+        node: 101
+        path: "eth1/1"
+    state: present
+
+- name: Update/Change Port - Fabric SPAN Session to Port Channel Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_port
+    destination_port_channel:
+      port_channel:
+        template: ansible_test
+        name: ansible_test_resource_pc_1
+    state: present
+
+- name: Update/Change Port Channel - Fabric SPAN Session to EPG Fabric SPAN Session
+  cisco.mso.ndo_fabric_span_session:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_test
+    name: ansible_test_pc
+    destination_epg:
+      epg:
+        schema: ansible_test
+        template: Template1
+        anp: Anp1
+        name: EPG1
+      destination_ip: "1.1.1.1"
+      source_ip_prefix: "2.2.2.2"
+    state: present
+
```

Comment:
Are the update name using UUID and query using UUID examples missing?

Reply:
Done!